### PR TITLE
feat(al2023): warn about bootstrap.sh usage

### PR DESCRIPTION
**Pull request by cartermckinnon**
**Description of changes:**

Adds more helpful logging to `/var/log/cloud-init-output.log` when AL2 user data is used on an AL2023 instance, using a fake `bootstrap.sh`.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: templates/al2023/runtime/rootfs/usr/bin/bootstrap.sh
Comment (ndbaker1) [inline templates/al2023/runtime/rootfs/usr/bin/bootstrap.sh:1]: we actually want this on the same path as al2 right? i.e. `/etc/eks/bootstrap.sh`?

Comment (cartermckinnon) [reply templates/al2023/runtime/rootfs/usr/bin/bootstrap.sh:1]: ah yep, forgot we put this there. makes so much sense for a script to be in `etc`, after all 😭

**Review by ndbaker1 — APPROVED** [review]

**Comment by s7an-it** [comment]
@cartermckinnon, what does it mean, does it menn bootstrap.sh can't be used and in the terraform eks module that option should be turned off, how can I pass custom script, using the cloudinit and multipart shell + nodeconfig?

**Comment by bryantbiggs** [comment]
What are you passing to the shell script?

**Pull request merged**